.buildkite/build.rayci.yml

```diff
@@ -0,0 +1,39 @@
+group: build
+steps:
+  - label: ":tapioca: build: wheel {{matrix}}"
+    instance_type: medium
+    commands:
+      - ./ci/build/build-manylinux-ray.sh
+      - ./ci/build/build-manylinux-wheel.sh {{matrix}}
+      - if [ "$BUILDKITE_PULL_REQUEST" != "false" ]; then exit 0; fi
+      - export PATH=/opt/python/cp38-cp38/bin:$$PATH
+      - pip install -q aws_requests_auth boto3
+      - ./ci/env/env_info.sh
+      # Upload to branch directory.
+      - python .buildkite/copy_files.py --destination branch_wheels --path ./.whl
+      # Upload to latest directory.
+      - if [ "$BUILDKITE_BRANCH" == "master" ]; then python .buildkite/copy_files.py --destination wheels --path ./.whl; fi
+    matrix:
+      - cp37-cp37m 1.14.5
+      - cp38-cp38 1.14.5
+      - cp39-cp39 1.19.3
+      - cp310-cp310 1.22.0
+      - cp311-cp311 1.22.0
+    depends_on: manylinux
+    job_env: manylinux
+
+  - label: ":tapioca: build: jar"
```

Comment:
why do you separate the jar?

I think the ray wheel needs to have the compiled java bits inside it too?

Reply:
that's right the ray wheel needs the java bit; both jar and wheel runs 'build-manylinux-ray.sh' script which build java and dashboard part; but the zipping them up into a bundle can be executed separately